documentation/source/simulator_support.rst

```diff
@@ -342,6 +342,7 @@ GHDL
 
     GHDL support in cocotb is experimental.
     Some features of cocotb may not work correctly or at all.
+    At least GHDL 2.0 is required.
```

Comment:
Earlier versions of GHDL do work, it was just that we added exemptions to get GHDL to pass the regression sometime before 2.0 released. So this sentence is not really correct.

I'd rather not make any promises about which versions of simulators re supported outside of "the latest release".

Reply:
Moved over to https://github.com/cocotb/cocotb/pull/2980/files#r878893191, this PR only contains some of the same commits for testing, but let's keep the discussion in the other PR.